content/docs/guides/branch-restore.md

```diff
@@ -74,26 +74,9 @@ Similar to the manual restore operation using the Neon Console and API described
 
 ### Time Travel Assist
 
-To help troubleshoot your data's history, you can use Time Travel Assist to connect to any selected point in time within your history retention window and then run queries against that connection. It's a good idea to run this kind of query to make sure you've targeted the correct restore point before you restore a branch.
```

Comment:
Maybe "To explore/view/query"? Troubleshooting your history made pause to think about what that meant.

Reply:
changed to "To help review..."